plugins/ruby/plugin.zsh

```diff
@@ -0,0 +1,72 @@
+# Color definitions
+GEOMETRY_COLOR_RUBY_RVM_VERSION=${GEOMETRY_COLOR_PROMPT:-white}
+
+GEOMETRY_RUBY_RVM_SHOW_GEMSET=${GEOMETRY_RUBY_RVM_SHOW_GEMSET:-true}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_RUBY_RVM_VERSION=${GEOMETRY_SYMBOL_RUBY_RVM_VERSION:-"◆"}
+GEOMETRY_RUBY_RVM_VERSION=$(prompt_geometry_colorize $GEOMETRY_COLOR_RUBY_RVM_VERSION $GEOMETRY_SYMBOL_RUBY_RVM_VERSION) 
+
+get_full_ruby_version() {
+  if (( $+commands[ruby] )) && [[ -z $GEOMETRY_RUBY_VERSION_FULL ]]; then
+      GEOMETRY_RUBY_VERSION_FULL="$(ruby -v)"
+  fi
+}
+
+get_full_rvm_version() {
+  if (( $+commands[rvm] )) && [[ -z $GEOMETRY_RVM_VERSION_FULL ]]; then
+      GEOMETRY_RVM_VERSION_FULL="$(rvm -v)"
+  fi
+}
+
+geometry_prompt_ruby_setup() {
+  get_full_ruby_version
```

Comment:
Right, but you run it again on `render`. And you have to do so because there might be several ruby versions in different projects, right? So running on `setup` isn't going to do much. Or am I understanding it wrong?